source/common/common/assert.h

```diff
@@ -176,8 +176,8 @@ bool shouldLogAndInvokeEnvoyBugForEnvoyBugMacroUseOnly(const char* filename, int
 // ENVOY_BUG logging and actions are invoked only on power-of-two instances per log line.
 #define _ENVOY_BUG_IMPL(CONDITION, CONDITION_STR, ACTION, DETAILS)                                 \
   do {                                                                                             \
-    if (!(CONDITION) &&                                                                            \
-        Envoy::Assert::shouldLogAndInvokeEnvoyBugForEnvoyBugMacroUseOnly(__FILE__, __LINE__)) {    \
+    if (!(CONDITION) && Envoy::Assert::shouldLogAndInvokeEnvoyBugForEnvoyBugMacroUseOnly(          \
+                            absl::StrCat(__FILE__, __LINE__))) {                                   \
```

Comment:
I think what Matt meant was to use preprocessor string concatenation so there's no work to be done at runtime. Also, you probably need a separator between the parts, like `:`, eg `__FILE__ ":" #__LINE__`

Reply:
There was a little hairiness in just using #__LINE__, but done (I think)

Reply:
Looks like about the expected amount of preprocessor hairiness.